static/js/lib/url_test.js

```diff
@@ -207,4 +208,18 @@ describe("url helper functions", () => {
       assert.ok(url.includes(postDetailURL(post.channel_name, post.id)))
     })
   })
+
+  describe("embedlyThumbnail", () => {
+    it("should return a good embedly url", () => {
+      const key = "embedlyKey"
+      const height = 100
+      const width = 200
+      const url = "http://www.fake.url/fake.html"
+      const embedlyUrl = embedlyThumbnail(key, url, height, width)
+      assert.ok(embedlyUrl.includes(`height=${height}`))
+      assert.ok(embedlyUrl.includes(`width=${width}`))
+      assert.ok(embedlyUrl.includes(`key=${key}`))
+      assert.ok(embedlyUrl.includes(`url=${encodeURIComponent(url)}`))
+    })
+  })
```

Comment:
might be good to just have a super basic test for the `blankThumbnailUrl` function too

Reply:
added a test